packages/b/backward-cpp/xmake.lua

```diff
@@ -19,6 +25,11 @@ package("backward-cpp")
     add_deps("cmake")
 
     on_install("linux", "mingw", "macosx", "windows", function (package)
+        if package:config("bfd") then
+            package:add("deps", "binutils")
```

Comment:
We need to dwonload/fetch deps beforce on_install. Move `package:add("deps", "binutils")` to on_load